cloudbaseinit/conf/default.py

```diff
@@ -162,6 +162,8 @@ def __init__(self, config):
                 default=[
                     'cloudbaseinit.metadata.services.httpservice.HttpService',
                     'cloudbaseinit.metadata.services'
+                    '.vmwareguestinfoservice.VMwareGuestInfoService',
+                    'cloudbaseinit.metadata.services'
```

Comment:
this metadata service should not be put as a default service, as it will degrade the performance (cloudbase-init runs mostly on openstack).